extensions/wikia/Email/Controller/WelcomeController.class.php

```diff
@@ -24,41 +25,72 @@ protected function getSubject() {
 	public function body() {
 		$this->response->setData( [
 			'salutation' => $this->getSalutation(),
-			'summary' => $this->getMessage( 'emailext-welcome-summary' )->text(),
-			'extendedSummary' => $this->getMessage( 'emailext-welcome-summary-extended' )->text(),
+			'summary' => $this->createSummary(),
 			'details' => $this->getDetailsList(),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "details" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)